src/core/flow.js

```diff
@@ -44,12 +47,23 @@ export function Flow({
   validator,
   processMove,
   optimisticUpdate,
+  canMakeMove,
 }) {
   if (!ctx) ctx = () => ({});
   if (!events) events = {};
   if (!init) init = state => state;
   if (!validator) validator = () => true;
   if (!processMove) processMove = state => state;
+  if (!canMakeMove) {
+    canMakeMove = (G, ctx, playerID) => {
+      const actionPlayers = ctx.actionPlayers || [];
+      return (
+        playerID === ctx.currentPlayer ||
+        actionPlayers.includes(playerID) ||
+        actionPlayers.includes('any')
```

Comment:
How about we get rid of the concept of `any` (it's a bit of a hack)? We can simulate it by just adding all possible playerID's to `actionPlayers`.